### ISSUE TITLE
fetch/DMS: investigate impact of low `kv.transaction.max_intents_bytes` settings 

### ISSUE THREAD

**Issue by ZhouXing19** [cockroachdb/molt#109]
From a customer ticket (mentioned in [internal link](https://cockroachlabs.slack.com/archives/C04TQ67V1JL/p1704478093406349?thread_ts=1704476193.188639&cid=C04TQ67V1JL)), we noticed AWS DMS stopped progressing, and in parallel CRDB logged:

```
a transaction has hit the intent tracking limit (kv.transaction.max_intents_bytes); is it a bulk operation? Intent cleanup will be slower. txn: "sql txn"
```

The solution in that ticket is to set a higher value for `kv.transaction.max_intents_bytes`. 

However, we did the following experiment with `molt fetch`, and cannot reproduce the same behavior. i.e. though the same logs are seen, the fetch attempt finished without error.

```
Schema: CREATE TABLE tbl1(id UUID PRIMARY KEY DEFAULT gen_random_uuid(), t TEXT)
Data load: 10m rows with t = 4-char string  or 20k rows with t = 200-char string
Target DB: SET CLUSTER SETTING kv.transaction.max_intents_bytes = 3;
Fetch command:

❯ go run . fetch \
  --source 'postgres://root@localhost:26257/defaultdb?sslmode=disable' \
  --target 'postgres://root@localhost:26258/defaultdb?sslmode=disable' \
  --table-filter 'tbl1' \
  --local-path /tmp/basic --local-path-listen-addr '0.0.0.0:9005' --live
```

We need to figure out if a low `kv.transaction.max_intents_bytes` value is really the cause of _DMS_ to exit / hang.

**Comment by ZhouXing19** [comment]
Closing this in favor of the jira ticket CC-26808
